src/sagemaker/serve/builder/jumpstart_builder.py

```diff
@@ -57,6 +57,8 @@ def __init__(self):
         self.mode = None
         self.model_server = None
         self.image_uri = None
+        self._is_custom_image_uri = False
+        self.vpc_config = None
```

Comment:
What's the added VPC config for?

Reply:
It's for feature parity. This config has already exposed to djl/tgi/torchserve/tri